src/Digdir.Domain.Dialogporten.Infrastructure/Persistence/Configurations/Dialogs/DialogEntityConfiguration.cs

```diff
@@ -22,5 +22,7 @@ public void Configure(EntityTypeBuilder<DialogEntity> builder)
         builder.HasIndex(x => x.ServiceResource);
         builder.HasIndex(x => x.Party);
         builder.HasIndex(x => x.Process);
+        builder.Property(x => x.IdempotentKey).HasMaxLength(36);
+        builder.HasIndex(x => new { x.Org, x.IdempotentKey }).IsUnique().HasFilter(null);
```

Comment:
```suggestion
        builder.HasIndex(x => new { x.Org, x.IdempotentKey }).IsUnique().HasFilter(@"\"{nameof(DialogEntity.IdempotentKey)}\" is not null");
```

Må oppdatere migrasonen etter denne endringen 🙂

Reply:
Er ikke dette default filteret? Er det da vits å ha `.HasFilter()`? Eller er det noe unntak for composite indexer

[Filter](https://learn.microsoft.com/en-us/ef/core/modeling/indexes?tabs=data-annotations#index-filter)
> When using the SQL Server provider EF adds an 'IS NOT NULL' filter for all nullable columns that are part of a unique index. To override this convention you can supply a null value.

Reply:
Vi bruker ikke SQL server provideren til EF. Vi bruker npgsql provideren fordi vi bruker postgres. Av det jeg kan se av migrasjonen du har lagt har det ikke blitt lagt til noe index filter. Ligger det en where clause på indexen dersom du går inn i lokal db etter å ha kjørt migrasjonen?